lib/flutter_camera_ml_vision.dart

```diff
@@ -152,18 +145,20 @@ class CameraMlVisionState<T> extends State<CameraMlVision<T>>
   }
 
   CameraValue get cameraValue => _cameraController?.value;
+
   ImageRotation get imageRotation => _rotation;
 
   Future<void> Function() get prepareForVideoRecording =>
       _cameraController.prepareForVideoRecording;
 
-  Future<void> startVideoRecording(String path) async {
+  Future<void> startVideoRecording() async {
     await _cameraController.stopImageStream();
-    return _cameraController.startVideoRecording(path);
+    return _cameraController.startVideoRecording();
   }
 
-  Future<void> stopVideoRecording() async {
-    await _cameraController.stopVideoRecording();
+  Future<void> stopVideoRecording(String path) async {
+    final file = await _cameraController.stopVideoRecording();
+    await file.saveTo(path);
```

Comment:
maybe we should let the user take care of this by returning file ?

Reply:
like it I'll change that!